pubsub.go

```diff
@@ -775,6 +784,12 @@ func (p *PubSub) handleIncomingRPC(rpc *RPC) {
 		}
 	}
 
+	// ask the router to vet the peer before commiting any processing resources
+	if !p.rt.AcceptFrom(rpc.from) {
+		log.Warningf("received message from router graylisted peer %s. Dropping RPC", rpc.from)
```

Comment:
should we do something if we get a bunch of these?

Reply:
we could blacklist i guess, but then we'd have to count stuff.